operations/power_management/Power_On_Compute_Cabinets.md

```diff
@@ -43,8 +43,12 @@ power-on command from Cray System Management \(CSM\) software.
 
 1. Use the System Admin Toolkit \(`sat`\) to power on liquid-cooled cabinets chassis and slots.
 
+   Note that the default timeout for powering the compute cabinets is 60 seconds, which is generally too low.
```

Comment:
@bklei, was this added due to CRAYSAT-1623? If so, it is not going to help with the timeout that occurs while waiting for the hms-discovery cronjob to be scheduled. I'm going to remove it when I rebase my PR #2841 on the latest main branch.